shotover/src/transforms/cassandra/sink_cluster/topology.rs

```diff
@@ -532,7 +534,7 @@ mod system_peers {
             ..
         })) = message.frame()
         {
-            return error.message == "unconfigured table peers_v2";
+            return error.ty == ErrorType::Invalid;
```

Comment:
Is this type guaranteed to be for this specific situation?
Or are we fine to try when this is detected regardless of the original cause?